src/app/helptext/storage/volumes/volume-list.ts

```diff
@@ -5,7 +5,7 @@ export default {
 unlockDialog_password_placeholder: T('Passphrase'),
 
 unlockDialog_recovery_key_placeholder: T('Recovery Key'),
-unlockDialog_recovery_key_tooltip: T('Click <b>Browse</b> to select a recovery key to\
+unlockDialog_recovery_key_tooltip: T('Click <b>Browse</b> to select a recovery key to \
```

Comment:
```suggestion
unlockDialog_recovery_key_tooltip: T('Unlock the pool with a recovery key file instead of a passphrase. Click <b>Browse</b> to select a recovery key file to \
```